src/lib.rs

```diff
@@ -820,6 +820,8 @@ macro_rules! __impl_bitflags {
                 result
             }
         }
+
+        __impl_arbitrary_for_bitflags!($BitFlags);
```

Comment:
```suggestion
        #[cfg(feature = "impl_arbitrary")]
        impl<'a> arbitrary::Arbitrary<'a> for $BitFlags {
            fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
                 Self::from_bits(u.arbitrary()?).ok_or_else(|| arbitrary::Error::IncorrectFormat)
            }
        }
```
There's no need for a separate helper macro afaict.